### PR TITLE
nifi 1.11.2

### PR DESCRIPTION
**Pull request by apiri**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/nifi.rb [hunk 1]
```diff
@@ -1,10 +1,8 @@
 class Nifi < Formula
   desc "Easy to use, powerful, and reliable system to process and distribute data"
   homepage "https://nifi.apache.org"
-  url "https://www.apache.org/dyn/closer.lua?path=nifi/1.11.1/nifi-1.11.1-bin.tar.gz"
-  mirror "https://archive.apache.org/dist/nifi/1.11.1/nifi-1.11.1-bin.tar.gz"
```
Comment (SMillerDev) [inline Formula/nifi.rb]: Please put the mirror back

**Comment by apiri** [comment]
Hi @SMillerDev!  Apologies for missing that.  Would a useful enhancement for the bump-formula-pr preserve/warn of the existence of mirrors or is there an expectation that the mirror is specified for each execution?

**Comment by SMillerDev** [comment]
Someone is actually working on that, it's just not merged yet.

**Comment by apiri** [comment]
thanks for the info!

**Comment by zbeekman** [comment]
Thanks for your contribution to Homebrew! 🎉 🥇 

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you @apiri!

**Comment by zbeekman** [comment]
Darn, I meant to squash the commits before pushing them.. Oh well, my bad.

**Pull request closed without merging**